docs/running_rune_with_occlum.md

```diff
@@ -4,26 +4,42 @@
 ## Requirements
 - Install [Intel SGX driver for Linux](https://github.com/intel/linux-sgx-driver#build-and-install-the-intelr-sgx-driver), required by Intel SGX SDK && PSW.
 - Install [enable_rdfsbase kernel module](https://github.com/occlum/enable_rdfsbase#how-to-build), allowing to use `rdfsbase` -family instructions in Occlum.
-- Assume the host system is CentOS 7.5.
-- You can also launch a CentOS 7.5 container as your host system with the following command:
-  ```shell
-  docker run -it --privileged --device /dev/isgx centos:7.5.1804
-  ```
-  If so, you need to run **another docker daemon** inside this CentOS 7.5 container. Please refer to [this guide](https://docs.docker.com/engine/install/centos) to install docker daemon, and type the following command to start dockerd.
-  ```shell
-  dockerd -b docker0 --storage-driver=vfs &
-  ```
+- Ensure that you have one of the following required operating systems:
+	- CentOS 7.5. You can also launch a CentOS 7.5 container as your host system with the following command:
```

Comment:
Don't use tab as indent

Reply:
Change to:

```
- Ensure that you have one of the following host systems:
  - CenOS 7.5
  - Ubuntu 18.04

  Note: You can launch a container corresponding to above host system.
  ```shell
  docker run -it --privileged --device /dev/isgx centos:7.5.1804
  ```
  or
  ```
  docker run -it --privileged --device /dev/isgx ubuntu:18.04
  ```

Reply:
Move the host system requirement beyond other items.